org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+                    }
+                    else
+                    {
+                        properties.Add( new HubspotPropertyUpdate() { property = "email", value = "" } );
+                    }
+
+                    // eNews Subscriber true or false
+                    string eNewsSub = ( person.Email != "" && person.Email != null && eNewsEmails.Contains( person.Email.ToLower() ) ) ? "true" : "false";
+                    properties.Add( new HubspotPropertyUpdate() { property = "enews_subscriber", value = eNewsSub } );
+                    Debug.WriteLine( "Patching: eNews: " + eNewsSub + " | " + person.Email );
+
+                    // Discpleship Step Path step completed date gathering
+                    Dictionary<int, string> stepsPath = new Dictionary<int, string>()
+                    {
+                        { 26, "baptism" },
+                        { 27, "life_groups" },
+                        { 29, "first_time_serving" }
+                    };
+                    foreach ( KeyValuePair<int, string> kvp in stepsPath )
+                    {
+                        var stepQuery = from s in _context.Steps
+                                       join pa in _context.PersonAliases on s.PersonAliasId equals pa.Id
+                                       where s.StepTypeId == kvp.Key && pa.Id == s.PersonAliasId && pa.PersonId == person.Id
+                                       select s.CompletedDateTime;
+                        var stepResult = stepQuery.FirstOrDefault().ToStringSafe();
+                        stepResult = stepResult != "" ? ConvertDate( ( DateTime ) stepResult.AsDateTime() ) : stepResult;
+                        Debug.WriteLine( "Discpleship Step Path : " + kvp.Value + " | completed date: " + stepResult );
+                        properties.Add( new HubspotPropertyUpdate() { property = kvp.Value, value = stepResult } );
+                    }
+
+                    //// First time Next Step Class Scheduled
+                    //var nscQuery = from h in _context.Histories
+                    //               where h.ValueName == "Next Step Class Scheduled" && h.EntityId == person.Id
+                    //               orderby h.Id ascending
+                    //               select h.NewValue;
+                    //var nscResult = nscQuery.FirstOrDefault().ToStringSafe();
+                    //properties.Add( new HubspotPropertyUpdate() { property = "next_steps_class_registration", value = nscResult } );
+                    //Debug.WriteLine( "First Time Next Step Class Scheduled : " + nscResult );
+
+
+                    try
+                    {
+                        //Update the Contact in Hubspot
+                        MakeRequest(current_id, url, properties, 0);
+                        //WriteToLog( string.Format( "    After Request: {0}", watch.ElapsedMilliseconds ) );
+                    }
+                    catch ( Exception err )
```

Comment:
The standard name for an Exception variable in a try/catch is `e`. Can you update this to follow the standard?